pkgs/by-name/te/testkube/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "testkube";
+  version = "2.1.56";
+
+  src = fetchFromGitHub {
+    owner = "kubeshop";
+    repo = "testkube";
+    rev = "v${version}";
+    sha256 = "sha256-P+A9lUMzQ3M0SEVZBMDSMj8S0uCsEhadv5vDRxbQORA=";
+  };
+
+  vendorHash = "sha256-44aIwddMH6CMfTno90xGkHgna4DO2Ii3KhpMwv6Zjmo=";
+
+  ldflags = [
+    "-X main.version=${version}"
+    "-X main.builtBy=nixpkgs"
+    "-X main.commit=v${version}"
+    "-X main.date=1970-01-01-00:00:01"
+  ];
+
+  doCheck = false;
+  subPackages = [ "cmd/kubectl-testkube" ];
+
```

Comment:
Would be better to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook, since you override version and skipped test.

Reply:
I just cannot get the versioncheckhook working with testkube. When running 'kubectl-testkube version' it displays some colored ASCII art before displaying the version so I think this might mess up the version parser.

Reply:
Could you try https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion?

Reply:
@Aleksanaa sorry I finally figured it out: 'kubectl-testkube version' command needs to reach out to 'https://api.github.com/repos/kubeshop/testkube/releases/latest' else it fails without displaying version. Thus I cannot use the versioncheckhook and testers.testVersion:

```
❯ nix-build --attr pkgs.testkube.passthru.tests                                       
this derivation will be built:
  /nix/store/yhly75lw5107lksdlh3yld4ykzyzn8p6-testkube-2.1.56-test-version.drv
building '/nix/store/yhly75lw5107lksdlh3yld4ykzyzn8p6-testkube-2.1.56-test-version.drv'...
kubectl-testkube version returned a non-zero exit code.

Gettong latest version (error: Get "https://api.github.com/repos/kubeshop/testkube/releases/latest": dial tcp: lookup api.github.com: no such host)
error: builder for '/nix/store/yhly75lw5107lksdlh3yld4ykzyzn8p6-testkube-2.1.56-test-version.drv' failed with exit code 1;
       last 3 log lines:
       > kubectl-testkube version returned a non-zero exit code.
       >
       > Gettong latest version (error: Get "https://api.github.com/repos/kubeshop/testkube/releases/latest": dial tcp: lookup api.github.com: no such host)
       For full logs, run 'nix log /nix/store/yhly75lw5107lksdlh3yld4ykzyzn8p6-testkube-2.1.56-test-version.drv'.
```

Do you know of a workaround for this? Thank you!

Reply:
I don't think there are either